src/hct_mis_api/apps/dashboard/templates/dashboard/dashboard.html

```diff
@@ -436,8 +408,18 @@ <h3 class="text-sm font-semibold">
                             `${decimalnumberFormatter(pendingPaymentsGroupUSD.sum)} USD`;
                         document.getElementById("total-amount-paid").innerHTML =
                             `${decimalnumberFormatter(totalSuccessfulUSD)} USD`;
-                        document.getElementById("total-amount-paid-local").innerHTML =
-                            `${decimalnumberFormatter(totalSuccessful)} ${localCurrency}`;
+                        let localCurrencyHtml = '';
+                        const currencyTotals = successfulPaymentsByCurrencyGroup.all();
+                        currencyTotals.forEach(item => {
+                            if (item.value.sum > 0 && item.key !== 'UNK') {
```

Comment:
what's UNK?

Reply:
Unknown currency. It's not supposed to happen.